### PR TITLE
qm-suspend: add page

### PR DESCRIPTION
**Pull request by BabaYaga1221**
Requested here - #8537

- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/qm-suspend.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# qm suspend
+
+> Suspends a virtual machine (VM) in the Proxmox Virtual Environment (PVE) cluster using the "qm" command-line tool.
+> More information: <https://pve.proxmox.com/pve-docs/qm.1.html>.
+
+- To specifies the ID of the virtual machine to suspend:
+
+`qm suspend -id 101`
```
Comment (einverne) [inline pages/linux/qm-suspend.md:8]: id should in {}

File: pages/linux/qm-suspend.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# qm suspend
+
+> Suspends a virtual machine (VM) in the Proxmox Virtual Environment (PVE) cluster using the "qm" command-line tool.
```
Comment (einverne) [inline pages/linux/qm-suspend.md:3]:
```suggestion
> Suspends a virtual machine (VM) in the Proxmox Virtual Environment (PVE) .
```

File: pages/linux/qm-suspend.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+> More information: <https://pve.proxmox.com/pve-docs/qm.1.html>.
+
+- To specifies the ID of the virtual machine to suspend:
```
Comment (einverne) [inline pages/linux/qm-suspend.md:6]:
```suggestion
- Suspend virtual machine by id:
```

**Review by einverne — CHANGES_REQUESTED** [review]
Hi thank you for the new page, I left some suggestions for you to take a look.

**Comment by BabaYaga1221** [comment]
@einverne Thanks sir, can you review now and suggest any other changes?

File: pages/linux/qm-suspend.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# qm suspend
+
+> Suspends a virtual machine (VM) in the Proxmox Virtual Environment (PVE).
+> More information: <https://pve.proxmox.com/pve-docs/qm.1.html>.
+
+- Suspend virtual machine by id:
+
+`qm suspend -{id} 101`
```
Comment (kbdharun) [inline pages/linux/qm-suspend.md:8]:
```suggestion
- Suspend virtual machine by id:

`qm suspend {{vm_id}} {{integer}}`
```
We follow a similar structure for other `qm` pages.

File: pages/linux/qm-suspend.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+- To skips the lock check when suspending the VM. Use with caution, as it may lead to data corruption in certain situations:
+
+`qm suspend --vmid 101 --skiplock`
```
Comment (kbdharun) [inline pages/linux/qm-suspend.md:12]:
```suggestion
- Skip the lock check when suspending the VM:

`qm suspend {{vm_id}} {{integer}} --skiplock`
```

File: pages/linux/qm-suspend.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+
+- To skips the lock check for storage when suspending the VM. Use with caution, as it may lead to data corruption in certain situations:
```
Comment (kbdharun) [inline pages/linux/qm-suspend.md:14]:
```suggestion
- Skip the lock check for storage when suspending the VM:
```

File: pages/linux/qm-suspend.md [hunk 7]
```diff
@@ -0,0 +1,16 @@
+# qm suspend
+
+> Suspends a virtual machine (VM) in the Proxmox Virtual Environment (PVE).
```
Comment (kbdharun) [inline pages/linux/qm-suspend.md:3]:
```suggestion
> Suspends a virtual machine (VM) in the Proxmox Virtual Environment (PVE).
> Use `--skiplock` and `--skiplockstorage` flags with caution, as they may lead to data corruption in certain situations.
```

File: pages/linux/qm-suspend.md [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+
+`qm suspend -{id} 101`
+
+- To skips the lock check when suspending the VM. Use with caution, as it may lead to data corruption in certain situations:
+
+`qm suspend --vmid 101 --skiplock`
+
+- To skips the lock check for storage when suspending the VM. Use with caution, as it may lead to data corruption in certain situations:
+
+`qm suspend --vmid 101 --skiplockstorage`
```
Comment (kbdharun) [inline pages/linux/qm-suspend.md:16]:
```suggestion
`qm suspend {{vm_id}} {{integer}} --skiplockstorage`
```

**Review by kbdharun — COMMENTED** [review]
Welcome to tldr and Thanks for your contribution. I have some suggestions for this page.

**Comment by BabaYaga1221** [comment]
@kbdharun Thanks for your review, sir. can you please retake a look and suggest any changes?

**Review by kbdharun — APPROVED** [review]

File: pages/linux/qm-suspend.md [hunk 9]
```diff
@@ -0,0 +1,17 @@
+# qm suspend
+
+> Suspends a virtual machine (VM) in the Proxmox Virtual Environment (PVE).
+> Use `--skiplock` and `--skiplockstorage` flags with caution, as they may lead to data corruption in certain situations.
+> More information: <https://pve.proxmox.com/pve-docs/qm.1.html>.
+
+- Suspend virtual machine by id:
```
Comment (acuteenvy) [inline pages/linux/qm-suspend.md:7]:
```suggestion
- Suspend a virtual machine by id:
```

**Comment by BabaYaga1221** [comment]
@acuteenvy append the changes as per your suggestion. Thanks

**Review by acuteenvy — APPROVED** [review]

**Comment by BabaYaga1221** [comment]
@einverne @acuteenvy @kbdharun can you merge this work? Thanks

**Review by einverne — APPROVED** [review]

**Pull request merged**